docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,160 @@
 ---
 title: Using the CLI
-sidebar_label: CLI Usage
+sidebar_label: Basic Usage
 slug: /cli-usage
 ---
 
-## Prerequisites
+You're probably here to run jobs (expressions) or workflows, which the CLI makes
+easy. Here's an outline of the basic usage scenarios
 
-1. Ensure you have a code editor installed on your machine (e.g.
-   [VS Code](https://code.visualstudio.com/),
-   [Sublime](https://www.sublimetext.com/))
+### Run a job
 
-2. Install NodeJs **v18 is the minimum version required**
+To run a single job, you must explicitly specify which adaptor to use.You can
+find the list of publicly available [adaptors here](/adaptors). See examples
+below
 
-   - To install a specific version of Node.js (in this case, version 18) on
-     Linux, Windows, or macOS, you can use a version manager such as nvm (Node
-     Version Manager) or any multiple runtime version manager eg:
-     [asdf](https://github.com/asdf-vm/asdf). These tools allow you to install
-     and switch between multiple versions of Node.js on the same machine. See
-     below for instructions for different operating systems.
-   - Read this article to learn how to install NodeJs in your machine
-     [kinsta.com/blog/how-to-install-node-js/](https://kinsta.com/blog/how-to-install-node-js/)
+> Pass the `-i` flag to auto-install that adaptor (it's safe to do this
+> redundantly).
 
-3. Have a basic understanding of OpenFn—check out jobs and adaptors, at least,
-   in the [Intro section](documentation/next) of this site.
-4. Install the OpenFn CLI with `npm install -g @openfn/cli`
+**Use a shorthand (e.g., `http`)**
 
-## CLI Usage - Key Commands
+```bash
+openfn path/to/job.js -ia http
+```
 
-You’ll learn about these commands in the following challenges, but please refer
-to this section for the key commands used in working with the CLI.
+**Use the full package name (e.g., `@openfn/language-http`)**
 
-### Check the version
+```bash
+openfn path/to/job.js -ia @openfn/language-http
+```
+
+**Add a specific version**
 
 ```bash
-openfn version
+openfn path/to/job.js -ia http@2.0.0
 ```
 
-### Get help
+**Pass a path to a locally installed adaptor**
 
 ```bash
-openfn help
+openfn path/to/job.js -a http=/repo/openfn/adaptors/my-http-build
 ```
 
-### Run a job
+### Write resulting state to disk
+
+After the job finishes, the CLI writes the resulting state to disk. By default,
+it creates an `output.json` next to the job file. You can specify custom paths
+for the output and state files.
+
+```bash
+openfn path/to/job.js -ia adaptor-name -o path/to/output.json -s path/to/state.json
+```
+
+### Pass state through stdin and return output through stdout
+
+Use `-S` and `-O` to pass state through stdin and return the output through
```

Comment:
I would say that -S is less useful for regular human usage because you have to escape the json. I wouldn't mention it here.

Reply:
I had the same feeling, i will take it out